frameworks/Java/blade/benchmark_config.json

```diff
@@ -0,0 +1,27 @@
+{
+  "framework": "blade",
+  "tests": [{
+    "default": {
+      "setup_file": "setup",
+      "json_url": "/json",
+      "db_url": "/db",
+      "query_url": "/db?queries=",
+      "plaintext_url": "/plaintext",
+      "port": 8080,
```

Comment:
Nit: the `README.md` states '9000' as the port, but here it's 8080.

Reply:
I saw most of the projects are 8080 ports, so the configuration of the 😄